src/cdk-experimental/public-api.ts

```diff
@@ -8,3 +8,4 @@
 
 export * from './version';
 export * from '@angular/cdk-experimental/scrolling';
+export * from '@angular/cdk-experimental/dialog';
```

Comment:
I think I want to change this to be the other way around- that we don't re-export anything through the root package. It will make the setup in google easier